src/utils.js

```diff
@@ -14,6 +14,7 @@ let atlasId = '0';
 let handlerEvents = {};
 let handlerKey = 0;
 let sendBeaconStatus = true;
+let targetWindow = window['parent'];
```

Comment:
I think it should be better to pass `targetWindow` as argument of constructor. `index.js` and `utils.js` has same code about `targetWindow` and if the implementation of `targetWindow` changes, we have to change both files. It possibly cause bugs.